quartz/src/main/java/org/quartz/simpl/RAMJobStore.java

```diff
@@ -779,10 +779,10 @@ public boolean removeCalendar(String calName)
         int numRefs = 0;
 
         synchronized (lock) {
-            for (TriggerWrapper trigger : triggersByKey.values()) {
-                OperableTrigger trigg = trigger.trigger;
-                if (trigg.getCalendarName() != null
-                        && trigg.getCalendarName().equals(calName)) {
+            for (TriggerWrapper wrapper : triggersByKey.values()) {
+                OperableTrigger trigger = wrapper.trigger;
+                if (trigger.getCalendarName() != null
+                        && trigger.getCalendarName().equals(calName)) {
```

Comment:
The outer thing is in fact a `wrapper` and not a `trigger` using `trigg` was misleading.